package fi.lipp.blog.model.exceptions

class UriIsBusyException : RuntimeException()